frontend/src/views/producer/dashboard/edition-form/Editor.jsx

```diff
@@ -237,6 +240,17 @@ class Editor extends React.Component {
       };
     }
   };
+
+  updateCoverImageName = imageurl => {
```

Comment:
the update and upload stuffs should not concern the form parent. It should be part of the CoverImage component which deals with the coverImage. You only patch the image url data to the form at the end of the upload cycle